tests/eosio.system_ram_tests.cpp

```diff
@@ -76,44 +76,44 @@ BOOST_FIXTURE_TEST_CASE( buy_sell_ram_validate, eosio_system_tester ) try {
    transfer( config::system_account_name, bob, core_sym::from_string("100.0000"), config::system_account_name );
    BOOST_REQUIRE_EQUAL( success(), buyrambytes( bob, bob, 10000 ) );
 
-   const char* expected_buyrambytes_return_data = R"=====(
+   std::string expected_buyrambytes_return_data = R"=====(
 {
    "payer": "alice",
    "receiver": "alice",
-   "quantity": "0.1462 TST",
+   "quantity": ")=====" + core_sym::from_string("0.1462").to_string() + R"=====(",
```

Comment:
These are not very pretty. It would be nice to be able to use `std::format` and `CORE_SYM_NAME`.

Reply:
You could add the include:

`#include <boost/algorithm/string.hpp>`

and then:

```c++
   std::string expected_buyramburn_return_data = boost::ireplace_all_copy<std::string>(R"=====(
{
   "payer": "alice",
   "receiver": "alice",
   "quantity": "1.0000 TST",
   "bytes_purchased": 68374,
   "ram_bytes": 86357,
   "fee": "0.0050 TST"
}
)=====", "TST", CORE_SYM_NAME);
```